src/playbooks/_database_connection/metadata.obsah.yml

```diff
@@ -0,0 +1,10 @@
+---
+variables:
+  foreman_database_host:
+    help: Hostname of the PostgreSQL server.
+  foreman_database_user:
+    help: Username for the Foreman DB.
+  foreman_database_password:
+    help: Password for the Foreman DB.
+  foreman_database_name:
+    help: Name of the Foreman DB.
```

Comment:
can you move the definition from `deploy` here, instead?

https://github.com/theforeman/foremanctl/blob/a56ab1daa893df57fda45cbb8a2d5c788b9da420/src/playbooks/deploy/metadata.obsah.yaml#L6-L39

Reply:
and then you load them into the playbook via
https://github.com/theforeman/foremanctl/blob/a56ab1daa893df57fda45cbb8a2d5c788b9da420/src/playbooks/deploy/deploy.yaml#L6-L10

Reply:
(you only need defaults and database, certificates and images aren't needed for checks (yet))